_data/downloads.yml

```diff
@@ -1,29 +1,21 @@
 stepOne:
   - os: linux
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <code class="hljs">$ curl -Lo cs https://git.io/coursier-cli-linux && chmod +x cs && ./cs setup </code> <br>
   - os: unix
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+            <p>Follow <a href="https://get-coursier.io/docs/cli-overview.html#install-native-launcher" target="_blank">the instructions to install the <code>cs</code> launcher</a> then run:</p>
+            <p><code>$ ./cs setup</code></p>
   - os: macos
-    text: "<p><code>java -version</code><span>(Make sure you have version 1.8.)</span></p>"
+    text: >-
+      <div class="highlight">
```

Comment:
Why `highlight` here but not in the linux and unix sections?

Reply:
It's a mistake